pkgs/applications/graphics/identity/default.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, rustPlatform
+, appstream-glib
+, blueprint-compiler
+, desktop-file-utils
+, gst_all_1
+, gtk4
+, libadwaita
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook4
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  pname = "identity";
+  version = "0.5.0";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "YaLTeR";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0l2v07pvcqi6mq8xw7n88jg0w03lj7cld5vm0r08i7mhrmavc4k4";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-5NUnrBHj3INhh9zbdwPink47cP6uJiRyzzdj+yiSVD8=";
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    blueprint-compiler
+    desktop-file-utils
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+  ] ++ (with rustPlatform; [
+    cargoSetupHook
+    rust.cargo
+    rust.rustc
+  ]);
+
+  buildInputs = [
+    gtk4
+    libadwaita
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-libav
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
+    description = "A program for comparing multiple versions of an image or video";
+    maintainers = with maintainers; [ paveloom ];
+    license = licenses.gpl3Plus;
+    platforms = platforms.linux;
+  };
```

```suggestion
  meta = {
    homepage = "https://gitlab.gnome.org/YaLTeR/identity";
    description = "A program for comparing multiple versions of an image or video";
    maintainers = with lib.maintainers; [ paveloom ];
    license = lib.licenses.gpl3Plus;
    platforms = lib.platforms.linux;
  };
```

Reasoning:
- https://github.com/atorres1985-contrib/nixpkgs/commit/451dfa6d84586b5cb83e6a7254689dff66b4523a